Alicerce/Sources/Protocols/ReusableView.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ReusableView.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 16/12/2016.
+//  Copyright © 2016 Mindera. All rights reserved.
+//
+
+import UIKit
+
+public protocol ReusableView: View {
+    static var reuseIdentifier: String { get }
+}
+
+public extension ReusableView where Self: UIView {
+    public static var reuseIdentifier: String { return "\(self)" }
+}
+
+public extension ReusableView where Self: UITableViewCell {
+    public init() {
```

Comment:
Perhaps we could have a `style` parameter defaulting to `.default`. What do you say?